boards/arm/arduino_zero/board.cmake

```diff
@@ -0,0 +1 @@
+include($ENV{ZEPHYR_BASE}/boards/common/openocd.board.cmake)
```

Comment:
add missing license/copyright info

Reply:
Done.  FWIW, it's rare for board.cmake or other short build files to have a copyright header.